Sources/SQLKit/Expressions/Queries/SQLCreateEnum.swift

```diff
@@ -1,25 +1,41 @@
-/// The `CREATE TYPE` command is used to create a new types in a database.
-///
+/// An expression representing a `CREATE TYPE` query. Used to create enumeration types.
+/// 
+/// ```sql
+/// CREATE TYPE "name" AS ENUM ('value1', 'value2');
+/// ```
+/// 
+/// This expression does _not_ check whether the current dialect supports separate enumeration types; users should
+/// take care not to use it with incompatible drivers.
+/// 
+/// > Note: As with ``SQLAlterEnum``, the full range of the `CREATE TYPE` query is not supported by this expression.
+/// 
 /// See ``SQLCreateEnumBuilder``.
 public struct SQLCreateEnum: SQLExpression {
-    /// Name of type to create.
+    /// The name for the created type.
     public var name: any SQLExpression
 
+    /// The enumeration values for the new type.
+    ///
+    /// Must contain at least one value.
```

Comment:
We don't. There are NUMEROUS other places where an array must have at least one value but this is never asserted; it is left to the underlying execution of the query against the database to report the error (since the "must" in this context means "otherwise invalid SQL will be generated").